simpledotorg/simple-server
# Show est population coverage totals column

**Story card:** [sc-10082](https://app.shortcut.com/simpledotorg/story/10082/update-to-coverage-section-to-include-a-totals-column-showing-the-total-estimate-population) Story to be created

This adds the additional total column for the coverage section.

## This addresses

This puts the design in line with the other sections in this card showing a percentage and a total.

## Test instructions

Check top level districts - this section is present in the 'Compare Performance' > `Hypertension indicators`

Please test with:
- empty estimated populations
- semi complete populations
- fully complete populations
- all populations entered as 0 for all regions

Please hover all cells during all stages to ensure correct wording on tooltip hovers

Please check the Coverage (?) tooltip is working as expected.
